src/contributing/index.md

```diff
@@ -1,13 +1,36 @@
 # Contributing
 
-There's more to running a distribution than just writing code. This section
-explains how to be an active part of Void.
+There's more to running a distribution than just writing code.
 
```

Comment:
This is, aesthetically a double paragraph still. I wonder if adding subheadings or just saying "To contribute software ports to ..." and "To contribute documentation to..." might help?

Reply:
The latter seems like a good approach .... Suggested exact wording to cover both?